js/views/reader_view.js

```diff
@@ -730,11 +726,6 @@ ReadiumSDK.Views.ReaderView = function(options) {
     function openPage(pageRequest, dir) {
 
         initViewForItem(pageRequest.spineItem, function(isViewChanged){
-
-            if(!isViewChanged) {
-                _currentView.setViewSettings(_viewerSettings);
-            }
-
```

Comment:
My point exactly: what about the `callback(false); return;` case?
(as per the code snippet quoted in my original message)
See:
https://github.com/readium/readium-shared-js/blob/feature/172-duplicate-pagination-changed/js/views/reader_view.js#L188